examples/basic.rs

```diff
@@ -44,15 +44,12 @@ impl EnvController for MyController {
 
 #[tokio::main]
 async fn main() {
-    let args: Vec<String> = env::args().collect();
```

Comment:
Remove unnecessary part, the basic should be as simple as possible.

Reply:
👍